users/vieting/experiments/switchboard/transducer/feat/experiments.py

```diff
@@ -936,9 +1050,9 @@ def run_scf_stage3():
         "label_scorer_type": "tf-ffnn-transducer",
         "epochs": [210],
     }
-    nn_system_stage2.returnn_configs["fullsum_scf_bs3k_v1_align-ctc-conf-e400"].recog_configs["recog"].config["network"]["output"][
-        "unit"
-    ]["label_context"][
+    nn_system_stage2.returnn_configs["fullsum_scf_bs3k_v1_align-ctc-conf-e400"].recog_configs["recog"].config[
+        "network"
+    ]["output"]["unit"]["label_context"][
         "from"
     ] = "output"  # no ILM, just SF
```

Comment:
If you already change this, you can also try to find a solution that is nicer including the rest of the command.